### ISSUE TITLE
Accessibility audit for Portal

### ISSUE THREAD

**Issue by lauracumming** [ocadotechnology/codeforlife-workspace#890]
We would like to complete an audit of the portal highlighting any accessibility issues and best practices that we should be adhering to or that could cause potential issues for our user groups.

There are tools, add ons and [standards](https://www.w3.org/WAI/standards-guidelines/wcag/) that can help you to analyse our website. Research best practice and apply this to your method.

Please also bear in mind the [Children's code](https://ico.org.uk/for-organisations/uk-gdpr-guidance-and-resources/childrens-information/childrens-code-guidance-and-resources/introduction-to-the-childrens-code/) when looking at this task.

- [ ] Write up findings in a document and create a list of design/research tasks

---

### Want to work on this issue?

[Code for Life](https://www.codeforlife.education/) welcomes and encourages community contributions!

If you would like to work on this issue, you can instruct @cfl-bot by commenting one of the following prompts:

|Prompt|Description|Arguments|Use When|
|---|---|---|---|
|*assign me*|Assigns you to this issue.|N/A|You want to work on this issue.|
|*unassign me*|Unassigns you from this issue.|N/A|You no longer want to work on this issue.|
|*ready for review*|Adds the *ready for review* label to this issue.|N/A|You've attached deliverables to this issue and you would like a CFL team member to review it.|
|*requires changes*|Removes the *ready for review* label from this issue.|N/A|You've realized that your deliverables are missing something.|
|*link pr \<number> \<repo?>*|Links a pull request to this issue.|**\<number>** The number of the PR to link.<br/>**\<repo?>** The repo the PR is in.* Defaults to this issue's repo.|You want to associate your code changes with this issue.|
|*unlink pr \<number> \<repo?>*|Unlinks a pull request from this issue.|**\<number>** The number of the PR to unlink.<br/>**\<repo?>** The repo the PR is in.* Defaults to this issue's repo.|You want to disassociate your code changes with this issue.|

*Only [CFL repos](https://github.com/orgs/ocadotechnology/repositories?q=codeforlife-) may be specified. You can optionally omit the "codeforlife-" prefix.

---

[Our Site](https://www.codeforlife.education/) | [Our Docs](https://docs.codeforlife.education/) | [Our Code](https://github.com/ocadotechnology/codeforlife-workspace)
